.github/workflows/container.yml

```diff
@@ -18,6 +18,11 @@ jobs:
         shell: bash
         run: echo "GITHUB_BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | tr / -)" >> $GITHUB_ENV
 
+      - name: Install qemu dependency
```

Comment:
Could we add a comment on why is this necessary?

Reply:
Done! :)